core-bundle/contao/templates/twig/backend/data_container/operations.html.twig

```diff
@@ -40,14 +40,27 @@
     {% elseif operation.href|default %}
         <li{% if menu and not operation.title|default %} data-contao--operations-menu-target="title"{% endif %}>
             {% set operation_attributes = attrs(operation.attributes|default)
-                .set('href', operation.href)
                 .set('title', operation.title|default, ((menu or showLabels) and operation.title|default != operation.label|default))
                 .addClass('has-icon', operation.icon|default)
             %}
-            <a{{ operation_attributes|raw }}>
-                {% if operation.icon|default %}{{ backend_icon(operation.icon, (menu or showLabels) ? '' : operation.title|default, operation.iconAttributes|default(null)) }}{% endif %}
-                {% if menu or showLabels %}{{ operation.label|default }}{% endif %}
-            </a>
+            {% if operation.method|default('GET') != 'GET' %}
+                <form action="{{ operation.href }}" method="POST">
+                    {% if operation.method != 'POST' %}<input type="hidden" name="_method" value="{{ operation.method }}">{% endif %}
+                    <input type="hidden" name="REQUEST_TOKEN" value="{{ contao.request_token }}">
+                    <button type="submit"{{ operation_attributes|raw }}>
+                        {% if operation.icon|default %}{{ backend_icon(operation.icon, (menu or showLabels) ? '' : operation.title|default, operation.iconAttributes|default(null)) }}{% endif %}
+                        {% if menu or showLabels %}{{ operation.label|default }}{% endif %}
+                    </button>
+                </form>
+            {% else %}
+                {% set operation_attributes = operation_attributes
+                    .set('href', operation.href)
+                %}
+                <a{{ operation_attributes|raw }}>
```

Comment:
```suggestion
                <a{{ operation_attributes }}>
```

Or inline it because there is only one attribute: 

```suggestion
                <a href="{{ operation.href }}">
```

Reply:
(L41 would be a good place to use the `attrs()`, though. :stuck_out_tongue:)

Reply:
But the `|raw` should be omitted, then.

Reply:
changed in 8beea2270b1fff33fb747b27320db1bd149347a2